api/app/controllers/spree/api/checkouts_controller.rb

```diff
@@ -84,6 +77,33 @@ def user_id
           params[:order][:user_id] if params[:order]
         end
 
+        def update_params
+          if update_params = massaged_params[:order]
```

Comment:
Yeah I'm with @jordan-brough and @jhawthorn on this. I think the assignment in the conditional makes a lot of sense here.